numpy/core/tests/test_scalarmath.py

```diff
@@ -144,8 +144,19 @@ def test_int_float_promotion_truediv(fscalar):
     # For int16, result has to be ast least float32 (takes ufunc path):
     assert (np.int16(1) / f).dtype == np.dtype("float32")
 
+def check_support_sve():
+    import subprocess
+    cmd = 'lscpu'
+    try:
+        subprocess.run(cmd, check=True)
+        process = (subprocess.Popen(cmd, stdout=subprocess.PIPE,
+                            shell=True).communicate()[0]).decode('utf-8')
+        return "sve" in process
+    except OSError:
+        return false
```

Comment:
Actually, same typo here.  Maybe move it together with the `IS_WASM` definition to not repeat this?

Reply:
Thank you very much.
I moved `check_support_sve()` functions to `numpy/testing/_private/utils.py`.

Reply:
OOps, made this comment a few days ago, but had it stored as pending review.  May just follow up today, though.

Thanks, now I notice that it is public, and since it feels a bit like a hack, I may want to rename it to _SUPPORTS_SVE for now (although happy to be convinced otherwise as soon as you need it in SciPy or so).
(Happy if others disagree, its just my gut feeling.)

You removed the except OSError:, maybe it is better to have it? I am wondering if it cannot fail on weirder setups.

Anyway, may just follow up on that myself quickly, thanks.

Reply:
@seberg 

Sorry for the late reply.
Changed to `_SUPPORTS_SVE`.
Changed the check method to use `OSError`.

I also reviewed the fujitsu compiler options.

If it looks good, put the commit together.

Thank you.

Reply:
Thanks, tests are failing now, do you know why?  I don't recall them failing before and am not aware what would have changed.

Reply:
Sorry, I fixed it so that the test passes.
Could you please check again?